_data-prepper/pipelines/configuration/sinks/opensearch.md

```diff
@@ -50,45 +50,82 @@ pipeline:
+`number_of_replicas` | No | Integer | The number of replica shards each primary shard should have on the destination OpenSearch server. For example, if you have 4 primary shards and set `number_of_replicas` to `3`, the index has 12 replica shards. This parameter is effective only when `template_file` is either explicitly provided in sink configuration or built-in. If this parameter is set, it would override the value in index template file. For more information, see [Create index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/).
+`distribution_version` | No | String  | Indicates whether the sink backend version is Elasticsearch 6 or later. `es6` represents Elasticsearch 6. `default` represents the latest compatible backend version, such as Elasticsearch 7.x, OpenSearch 1.x, or OpenSearch 2.x. Default is `default`.
+`enable_request_compression` | No | Boolean | Whether to enable compression when sending requests to OpenSearch. When `distribution_version` is set to `es6`, default is `false`. For all other distribution versions, default is `true`.
+`action` | No | String | The OpenSearch bulk action to use for documents. Must be one of `create`, `index`, `update`, `upsert`, or `delete`. The default value is `index`.
+`actions` | No | List | A [list of actions](#actions) that can be used as an alternative to `action` that reads as a switch case statement which conditionally determines the bulk action to take for an event. 
+`flush_timeout` | No | Long | A long class that contains the duration in milliseconds to try packing a bulk request up to the `bulk_size` before flushing the request. If this timeout expires before a bulk request has reached the `bulk_size`, the request will be flushed. Set to `-1` to disable the flush timeout and instead flush whatever is present at the end of each batch. Default is `60,000` or one minute.
+`normalize_index` | No | Boolean | If true, the OpenSearch sink will try to make dynamic index names. Index names with format options specified in `${})` valid according to the [index naming restrictions]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#index-naming-restrictions). Any invalid characters will be removed. Default value is false.
+`routing` | No | String | A string which is used as a hash for generating the `shard_id` for the document when it is stored in the OpenSearch. Each incoming record is searched for this field. When present, the string is used as the routing field for the document. When not present, the default routing mechanism of using `document_id` is used by OpenSearch when storing the document. Supports formatting with fields in the Events and [Data Prepper expressions]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/expression-syntax/), such as `${/my_field}-test-${getMetadata(\"some_metadata_key\")}`
```

Comment:
Should a noun follow "hash" (value? function?)? Can the first sentence just be "A string used as a hash [noun] for generating the `shard_id` for a document stored in OpenSearch"?

Reply:
How about starting the sentence with "A hash string?"

Reply:
No noun per discussion.